### PR TITLE
Cache city (and shows and fairs) data for one day

### PR DESCRIPTION
**Pull request by alloy**
Closes https://artsyproduct.atlassian.net/browse/LD-283

#skip_new_tests

### CODE REVIEW

File: Pod/Classes/Data/ARCity+GeospatialAdditions.m [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#import "ARCity+GeospatialAdditions.h"
+#import "ARCity.h"
+
+@implementation ARCity (GeospatialAdditions)
+
++ (nullable ARCity *)cityNearLocation:(CLLocation *)location;
+{
+    static const NSInteger CITY_RADIUS_M = 100 * 1000; // 100km
+
+    ARCity *closestCity = [[[ARCity cities] sortedArrayUsingComparator:^NSComparisonResult(ARCity *_Nonnull lhs, ARCity *_Nonnull rhs) {
```
Comment (ashfurrow) [inline Pod/Classes/Data/ARCity+GeospatialAdditions.m]: Nice cleanup on the types 👍

File: Pod/Classes/Data/ARCity.m [hunk 2]
```diff
@@ -1,39 +1,43 @@
 
-#import "ARCity.h"
+/**
+ * Do NOT alter this file directly! It is auto-generated by the `./scripts/generate-cities-objc.ts` script.
```
Comment (ashfurrow) [inline Pod/Classes/Data/ARCity.m]: Nice!

File: src/lib/Scenes/Map/GlobalMap.tsx [hunk 3]
```diff
@@ -420,12 +420,5 @@ export const GlobalMapContainer = createRefetchContainer(
         }
       }
     }
-  `,
-  graphql`
-    query GlobalMapRefetchQuery($near: Near!, $maxInt: Int!) {
-      viewer {
-        ...GlobalMap_viewer @arguments(near: $near, maxInt: $maxInt)
-      }
-    }
```
Comment (alloy) [inline src/lib/Scenes/Map/GlobalMap.tsx]: @l2succes I also went ahead and removed this.

https://artsy.slack.com/archives/CFNFWD5F0/p1551809664119900

**Comment by ashfurrow** [comment]
Since we added a new file, we need to bust the CI cache. I've done that in this commit: 
https://github.com/artsy/emission/pull/1389/commits/ffcc018b0b729fb77cfb52c5dbc0fb243168c392 We can merge on green.

File: src/lib/Scenes/Map/MapRenderer.tsx [hunk 4]
```diff
@@ -3,36 +3,40 @@ import { defaultEnvironment } from "lib/relay/createEnvironment"
...
+        if (mapProps) {
+          return <GlobalMap {...mapProps} {...props} />
         }
-
+        // TODO: Render some loading view
         return null
       }}
+      cacheConfig={
+        {
+          emissionCacheTTL: 86400, // 60 * 60 * 24 = 1 day
```
Comment (l2succes) [inline src/lib/Scenes/Map/MapRenderer.tsx]: 👍 glad that worked

**Pull request merged**